account-kit/react/src/components/auth/sections/EmailAuth.tsx

```diff
@@ -5,12 +5,12 @@ import { useAuthenticate } from "../../../hooks/useAuthenticate.js";
 import { useSigner } from "../../../hooks/useSigner.js";
 import { ChevronRight } from "../../../icons/chevron.js";
 import { MailIcon } from "../../../icons/mail.js";
+import { ls } from "../../../strings.js";
```

Comment:
where is this used?

Reply:
In the component itself, this moved because I ran the linter which re-ordered imports